spec/helpers/sessions.rb

```diff
@@ -0,0 +1,21 @@
+module SessionHelpers
+
+  def sign_up(user)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for sign_up is too high. [17/15]